### PR TITLE
Added GET endpoints for Connectors & Orchestrators

### PR DESCRIPTION
**Pull request by LotuxPunk**

### CODE REVIEW

File: packages/ldes-manager/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "ldes-manager",
```
Comment (DemianD) [inline packages/ldes-manager/package.json]:
```suggestion
  "name": "@ldes/ldes-manager",
```

File: packages/ldes-manager/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  "version": "0.0.0",
+  "description": "> TODO: description",
+  "homepage": "https://github.com/osoc21/ldes2service#readme",
+  "license": "ISC",
```
Comment (DemianD) [inline packages/ldes-manager/package.json]:
```suggestion
  "license": "MIT",
```

File: packages/ldes-manager/orm/sequelize.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+"use strict";
```
Comment (DemianD) [inline packages/ldes-manager/orm/sequelize.js]: You apparantly also pushed the `.js` files. You only have to commit the `.ts` files

Comment (LotuxPunk) [reply packages/ldes-manager/orm/sequelize.js]: Apparently js files outside bin or lib aren't ignored. Fixed that just now !

**Comment by LotuxPunk** [comment]
packages/ldes-manager/lib/QueryService.ts
packages/ldes-manager/lib/Manager.ts
packages/ldes-manager/bin/ldes-manager.ts

I don't know if we should get rid of these files. @D34DPlayer made those, but I feel like I'd not used them.

**Pull request merged**